### DISCUSSION TITLE
187d000c-32b3-4c5c-b45c-813611a2a5e0

### DISCUSSION THREAD

**Discussion by giscus** [ChooSeoyeon/blog-comment#16] [General]
# 187d000c-32b3-4c5c-b45c-813611a2a5e0

본 게시글은 하단 책을 읽고 학습한 내용을 제 생각으로 요약, 정리한 글입니다.

https://choo.oopy.io/187d000c-32b3-4c5c-b45c-813611a2a5e0

**Comment by andole98** [comment]
critical region보다 critical section이라는 용어가 더 일반적인것 같네요.
